applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_solver.py

```diff
@@ -96,8 +56,9 @@ def __init__(self, model, custom_settings):
             warning += 'from the "solver settings" if you dont use this wrapper, this check will be removed soon!\n'
             KratosMultiphysics.Logger.PrintWarning("Time integration method", warning)
 
-        # Overwrite the default settings with user-provided parameters.
-        self.settings.ValidateAndAssignDefaults(default_settings)
+        self._validate_settings_in_baseclass=True # To be removed eventually
```

Comment:
it is used like this, eventually this can be removed again after the derived solvers are updated